Add some information in doc for sirism (#128)
hove-io/forseti: internal/departures/readme.md

- `--departures-service-refresh` The refresh time between 2 request of the  external service(Required for `rennes`)
- `--departures-token` The token for the external service (Required for `rennes`)
- `--departures-notifications-stream-name` The name of the AWS Kinesis Data Stream (Required for `sirism`)
- `--departures-status-stream-name` The name of the AWS Kinesis Status Stream (Required for `sirism`)

> - `--departures-notifications-reload-period` The period since which the data records are being reloaded  (Optional but only for
> `sirism`)
> - `--departures-status-reload-period` The period since which the status records are being reloaded  (Optional but only for
> `sirism`)

Reviewer: ```suggestion
- `--departures-notifications-reload-period` The period from which the data records are reloaded  (Optional but only for
`sirism`)
- `--departures-status-reload-period` The period from which the status records are reloaded  (Optional but only for
`sirism`)
```

Author: :+1:
